README.md

```diff
@@ -152,6 +152,15 @@ This happens when a new import is added. Run `dep ensure` to install the missing
 
 As `dep status` suggests, run `dep ensure` to update your lockfile. Then run `dep status` again, and the lock mismatch should go away.
 
+### Visualizing dependencies
+
+Generate a visual representation of the dependency tree by piping the output of `dep status -dot` to [graphviz](http://www.graphviz.org/).
+```
+$ brew install graphviz
+$ dep status -dot | dot -T png | open -f -a /Applications/Preview.app
+```
+<p align="center"><img src="docs/img/StatusGraph.png"></p>
+
```

Comment:
Will do. I've done it on Linux, @carolynvs has done it on macOS. Do we know if it works on Windows?